app/src/models/init-models.ts

```diff
@@ -520,6 +520,8 @@ export function initModels(sequelize: Sequelize) {
     as: "cities",
     foreignKey: "userId",
     otherKey: "cityId",
+    onDelete: "CASCADE",
+    onUpdate: "CASCADE",
```

Comment:
Are all of these changes in the init models file in sync with the changes to the foreign key constraints?